include/crocoddyl/core/actions/unicycle.hpp

```diff
@@ -42,6 +42,9 @@ class ActionModelUnicycleTpl : public ActionModelAbstractTpl<_Scalar> {
   const Vector2s& get_cost_weights() const;
   void set_cost_weights(const Vector2s& weights);
 
+  Scalar get_dt() const;
+  void set_dt(Scalar dt);
```

Comment:
It would be good if this is `const`, i.e.,
```cpp
void set_dt(const Scalar dt);
```

I apologize for not being clear enough